hw/hw_04_task_02.py

```diff
@@ -0,0 +1,11 @@
+import urllib.request
+
+
+# flake8: noqa: S310
+def count_dots_on_i(url: str) -> int:
+    try:
+        page = urllib.request.urlopen(url)
+        data = page.read().decode()
+        return sum(1 for symb in data if symb == "i")
+    except:
```

Comment:
It is a bad practice to catch all errors.

You catch both networks and not network errors.   Check the documentation and do some experiments to reproduce this type of errors.